tests/store/model_registry/test_file_store.py

```diff
@@ -749,15 +771,109 @@ def search_versions(filter_string):
         name=mv1.name, version=mv1.version, description="Online prediction model!"
     )
 
-    mvds = store.search_model_versions(f"run_id = '{run_id_1}'")
-    assert len(mvds) == 1
+    mvds = store.search_model_versions("run_id = '%s'" % run_id_1, max_results=10)
+    assert 1 == len(mvds)
```

Comment:
```suggestion
    assert len(mvds) == 1
```

The value to check should be on the left-hand side.